core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -1140,6 +1156,23 @@ final class ShuffleBlockFetcherIterator(
     }
     removedChunkIds
   }
+
+  private[storage] def logFetchIfSlow(
+    durationMs: Long,
+    totalBlockSize: Long,
+    blockCount: Int,
+    blockManagerId: BlockManagerId): Unit = {
+    val transferBps = if (durationMs > 0) {
```

Comment:
minor nit: indent the parameter list by 4 spaces

Reply:
Fixed